pom.xml

```diff
@@ -384,7 +384,7 @@
         <version.org.apache.avro>1.11.0</version.org.apache.avro>
         <version.org.apache.cxf>3.5.2</version.org.apache.cxf>
         <version.org.apache.cxf.xjcplugins>3.3.1</version.org.apache.cxf.xjcplugins>
-        <version.org.apache.ds>2.0.0-M24</version.org.apache.ds>
+        <version.org.apache.ds>2.0.0.AM26</version.org.apache.ds>
```

Comment:
I think it would be good to split out the changes related to this component upgrade into a separate commit in this PR. We actually created an issue a little while ago to track this:

https://issues.redhat.com/browse/WFLY-14375

Reply:
Ah! Ok @fjuma! Then I'm going to separate the upgrade part to other PR (WFLY-14375) and this one will be updated later only with the new test classes. I didn't know there was another JIRA for the apache-ds upgrade. Converting this to draft until the upgrade is merged.